library/std/src/env.rs

```diff
@@ -799,7 +799,7 @@ impl DoubleEndedIterator for Args {
 #[stable(feature = "std_debug", since = "1.16.0")]
 impl fmt::Debug for Args {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        f.debug_struct("Args").field("inner", &self.inner.inner.inner_debug()).finish()
+        f.debug_struct("Args").field("inner", &format_args!("{:?}", self.inner.inner)).finish()
```

Comment:
Why the `format_args` instead of just passing `&self.inner.inner` to `field()`?

Reply:
That's better, I forgot `field` already takes `&dyn Debug`.